language/move-prover/spec-lang/src/env.rs

```diff
@@ -342,6 +342,12 @@ impl GlobalEnv {
         self.error_with_notes(loc, msg, vec![]);
     }
 
+    /// Adds an warning to this environment.
```

Comment:
Isn't it "a warning"? I'm myself all the time confused.

Reply:
Ah it was a copy/paste mistake! Original was "Adds an error"